src/vcpkg/export.cpp

```diff
@@ -485,12 +485,12 @@ namespace vcpkg::Export
     static void print_next_step_info(const Path& prefix)
     {
         const auto cmake_toolchain = prefix / "scripts/buildsystems/vcpkg.cmake";
-        const CMakeVariable cmake_variable = CMakeVariable("CMAKE_TOOLCHAIN_FILE", cmake_toolchain.generic_u8string());
+        CMakeVariable cmake_variable{"CMAKE_TOOLCHAIN_FILE", cmake_toolchain.generic_u8string()};
```

Comment:
The changes in this file seem unmotivated? `print2` has no reason to eat the string.